src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/ServiceBusConfiguration.java

```diff
@@ -44,4 +59,32 @@ public ServiceBusHelper notificationsQueueHelper(
             objectMapper
         );
     }
+
+    @Bean(name = "notifications-read")
+    @ConditionalOnProperty(prefix = "queues.read-notifications", name = "enabled")
+    public IQueueClient notificationsQueueReader(
```

Comment:
will this bean be used in next PR?

Reply:
reader will be enabled in aks and functional test should assert notification has been sent. i think i need help in setting up configs though